### PR TITLE
Allows lollipops to be worn in the mask slot, adds BoBo's pizza

### PR DESCRIPTION
**Pull request by bruhmogus**
# Description

Allows the lollipop food item to be worn in the mask slot since you can do that in real life and it looks cool. Also adds BoBo's Pizza, containing all 231 reagents I could find. I know I missed some but can't find them anywhere. Maintainers are welcome to add them as they find them. The pizza is admeme only.

---

# TODO

- [x] Add lollipops in mask slot
- [x] Add BoBo's pizza
- [x] Add BoBo's pizza slice

---

None. No testing. Just suffering.

---

# Changelog

:cl:
- add: Added Bo Bo's pizza
- tweak: Allowed the lollipop food item to be worn in the mask slot for realism and style.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml [hunk 1]
```diff
@@ -534,3 +534,973 @@
         - ReagentId: Vitamin
           Quantity: 1
 # Tastes like stale crust, rancid cheese, mushroom.
+
+- type: entity
+  name: Bo Bo pizza
+  parent: FoodPizzaBase
+  id: FoodPizzaBoBo
+  description: PIZZA WITH OVER 137 CHEMICALS IN IT!
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml:542]:
```suggestion
  description: PIZZA WITH OVER 137 CHEMICALS IN IT! OVER 1000 TOTAL CHEMICAL VOLUME!
```

File: Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/candy.yml [hunk 2]
```diff
@@ -22,6 +22,9 @@
   - type: Sprite
     sprite: Nyanotrasen/Objects/Consumable/Food/candy.rsi
     state: lollipop
+  - type: Clothing
+    slots: [mask]
+    sprite: Objects/Consumable/Smokeables/Cigarettes/cigarette.rsi
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/candy.yml:27]: this should probably not use the cigarette sprite
i feel the two items are distinct enough

**Review by TheDoctor1977 — CHANGES_REQUESTED** [review]
oughhhhhhhhhhhhhhhhhhhhhh

**Review by TheDoctor1977 — APPROVED** [review]
do not merge this yet 

let eris give this a look for the test fails

**Comment by bruhmogus** [comment]
unique sprites added anyway since i realised cigarettes werent pure white, revert to cigarette if these dont work

**Comment by Erisfiregamer1** [comment]
----> Robust.Shared.Prototypes.UnknownPrototypeException : Unknown ReagentPrototype prototype: CryptoBiolin

Test fails are your fault, sweep them and fix any related issues. Merge blocked until done.

**Comment by Erisfiregamer1** [comment]
Also **for the love of god** add some protection against that **certain pizza** being grinded. Not for balance, for the server's health.

**Comment by bruhmogus** [comment]
> Also **for the love of god** add some protection against that **certain pizza** being grinded. Not for balance, for the server's health.

and how do you propose i do that

**Comment by bruhmogus** [comment]
> > Also **for the love of god** add some protection against that **certain pizza** being grinded. Not for balance, for the server's health.
> 
> and how do you propose i do that

nevermind, pizza cant be put into grinders, so we're chillin

**Comment by bruhmogus** [comment]
all test fails seem unrelated now

**Comment by Erisfiregamer1** [comment]
yeah

**Pull request merged**